sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/forecast/forecasting_script.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
We do not use this script in the notebook, let us remove it.

Reply:
done